app.js

```diff
@@ -0,0 +1,157 @@
+$(document).ready(() => {
+  $('.search-trips').on('click', function() {
+
+    $.get('https://trektravel.herokuapp.com/trips',
+    response => {
+      console.log('success!');
+      //console.log(JSON.stringify(response));
+
+      let tableRows = '';
+      response.forEach(function(trip) {
+        tableRows += `<tr data-id=${trip.id}>`
+        for (let attr in trip) {
+          tableRows += '<td>' + trip[attr] + '</td>'
+        }
+        tableRows += '</tr>'
+      })
+
+      $('#trips table').show().append(tableRows);
+
+    })
+    .fail(function(response) {
+      //{"readyState":0,"status":0,"statusText":"error"}
+      // console.log('what returns '+ JSON.stringify(response));
+      $('#error').append("Content cannot be retrieved");
+      console.log('failure');
+    })
+    .always(function(){
+      console.log('always even if we have success or failure');
+    });
+
+    // FUNCTION FOR AJAX REQUEST AND RESPONSE FOR A SPECIFIC TRIP
+    let loadTrip = function(id) {
+      $('#trip').html('');
+      $.get(`https://trektravel.herokuapp.com/trips/${id}`,
+        (response) => {
+          // console.log('this is' + response[name]);  //why is this returning object object
+          let tripInfo = `
+
+          <p> Id: ${response.id} </p>
+          <p> Continent: ${response.continent} </p>
+          <p> About: ${response.about} </p>
```

Comment:
You should wrap this trip description in some sort of sectioning tag. Even dynamically generated content has to be semantic!